testing/typing_checks.py

```diff
@@ -4,11 +4,13 @@
 none of the code triggers any mypy errors.
 """
 import contextlib
+import sys
 from typing import Optional
 
 from typing_extensions import assert_type
 
 import pytest
+from _pytest.monkeypatch import MonkeyPatch
```

Comment:
In `typing_checks` we try to stick to the public API, so should use `pytest.MonkeyPatch` instead of `_pytest.monkeypatch.MonkeyPatch`.